### DISCUSSION TITLE
Named export 'TailwindPagination' not found. Nuxt 3 on Vercel

### DISCUSSION THREAD

**Discussion by partenit** [gilbitron/laravel-vue-pagination#154] [Help & support]
working without error locally on dev server
but after deploy to Vercel error:

500
Named export 'TailwindPagination' not found. The requested module 'laravel-vue-pagination' is a CommonJS module, which may not support all module.exports as named exports. CommonJS modules can always be imported via the default export, for example using: import pkg from 'laravel-vue-pagination'; const { TailwindPagination } = pkg;

my code:
![image](https://user-images.githubusercontent.com/1776676/203813637-f7ba42d4-0a2b-4dca-bd31-5028a970775f.png)

**Comment by gilbitron** [comment]
If it works locally, then I'd guess it's an issue with your deployment instead of an issue with the package itself.

**Comment by mrmt00** [comment]
This solved my issue:

```
import * as pkg from 'laravel-vue-pagination';
const { TailwindPagination } = pkg;
```
